.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,61 @@
-**Please provide the following information**
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED AS INCOMPLETE
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following listed requirements below,
+put an "x" (without the quotes) between the square brackets.
```

Comment:
That's excessive IMO. Obviously i can't talk for everyone but i would never do this no matter what. It would already be a hassle to click some big button "I agree to the terms and conditions" but having to hunt down multiple positions in text editing to do it? Absolutely unacceptable.

Reply:
You don't have to "hunt down" it's right below it. I don't think it is unacceptable,it is relatively common. Most notably, big projects here on github such as uBlock Origin, electron, angular.js and youtube-dl use it, and it doesn't prevent even the most layman users to post issues. Instead, it forces people to think about what they read and helps weed out low effort issue reports.

I am interested in hearing more opinions though. Is this really too much?

Reply:
I assume this comment was made under now-invalid assumptions (https://github.com/qbittorrent/qBittorrent/pull/11896#issuecomment-576386960), so I guess this is resolved.